profiling-ffi/src/crashtracker/datatypes.rs

```diff
@@ -325,12 +327,15 @@ impl<'a> TryFrom<&StackFrame<'a>> for datadog_crashtracker::StackFrame {
             }
             Some(vec)
         };
+        let relative_address = to_hex(value.relative_address);
```

Comment:
Is this valid to assume a relative offset of `0` is `None`? Or do we need to be more clever here?

Reply:
the term relative is ambiguous, as it can have different meanings.
we use `elf`, blazesym uses `normalized`. I can imagine you are aiming for cross compatibility ?
0 for an elf address is not possible (there are headers)